content/en/docs/tasks/administer-cluster/upgrade-downgrade/kubeadm-upgrade-1-11.md

```diff
@@ -0,0 +1,303 @@
+---
+reviewers:
+- pipejakob
+- luxas
+- roberthbailey
+- jbeda
+title: Upgrading kubeadm clusters from v1.10 to v1.11
+content_template: templates/task
+---
+
+{{% capture overview %}}
+
+This guide is for upgrading `kubeadm` clusters from version 1.10.x to 1.11.x, as well as 1.10.x to 1.10.y and 1.11.x to 1.11.y where `y > x`.
```

Comment:
We removed the possibility for kubeadm v1.11.0 to upgrade from v1.10.3 to v1.10.4 right b/c of the config stuff right?
Then this should be changed accordingly